jsapi/src/main.js

```diff
@@ -26,3 +29,95 @@ lux.Map = function(options) {
   goog.base(this, options);
 };
 goog.inherits(lux.Map, ol.Map);
+
+/**
+ * Show a marker on the map at the given location.
+ * @param {luxx.MarkerOptions} options Config options
+ * @export
+ */
+lux.Map.prototype.showMarker = function(options) {
+  var element = goog.dom.createDom(goog.dom.TagName.DIV);
+  var image = goog.dom.createDom(goog.dom.TagName.IMG);
+  image.src = options.iconURL ||
+      'http://openlayers.org/en/master/examples/data/icon.png';
+  element.appendChild(image);
+  var position = options.position || this.getView().getCenter();
+  this.addOverlay(new ol.Overlay({
+    element: element,
+    position: position,
+    positioning: options.positioning || 'center-center'
+  }));
+
+  if (options.autoCenter) {
+    this.getView().setCenter(position);
+  }
+
+  if (options.html) {
+    var popup;
+    var showPopupEvent = options.hover ?
+        ol.events.EventType.MOUSEMOVE : ol.events.EventType.CLICK;
+    ol.events.listen(element, showPopupEvent, (function() {
+      if (!popup) {
```

Comment:
Actually, the popup is (supposed to be) destroyed each time it's closed. Now I see that it would be better not to recreate i but to re-add it instead.